data/20-gnome-initial-setup.rules

```diff
@@ -21,8 +22,16 @@ polkit.addRule(function(action, subject) {
                          action.id.indexOf('org.freedesktop.realmd.') === 0 ||
                          action.id.indexOf('org.freedesktop.RealtimeKit1.') === 0);
 
+    var setupPrograms = [
+        '/usr/lib/gnome-initial-setup/eos-setup-live-user',
+        '/usr/libexec/eos-setup-demo-user',
```

Comment:
Okay, I've changed this to use `.policy` files and just keep everything under the `com.endlessm.DemoMode.` namespace